lib/jsdom/living/nodes/SVGTests-impl.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
```

Comment:
Should SVGTests be located in `svg/` instead of `nodes/`?

Reply:
It was originally in `svg/`, but it is only used as a mixin for other elements, and is therefore treated similarly as such interfaces in HTML.